tools/qt5_to_qt6_country_code_table.py

```diff
@@ -28,7 +28,7 @@ def parse_enum_from_header(header):
             continue
         matches = assign_pattern.match(assignment)
         if not matches:
-            raise Exception("unable to match line: %r" % assignment)
+            raise ValueError(f"unable to match line: {assignment!r}")
```

Comment:
Note to self: Check what !r means

Reply:
It's for `repr` and is equivalent to `%r` in old style